server/src/main/java/io/spine/server/bus/Bus.java

```diff
@@ -71,18 +88,106 @@ public void unregister(D dispatcher) {
     /**
      * Posts the message to the bus.
      *
-     * @param message the message to post
-     * @param responseObserver the observer to receive outcome of the operation
+     * <p>Use the {@code Bus} class abstract methods to modify the behavior of posting.
+     *
+     * @param message  the message to post
+     * @param observer the observer to receive outcome of the operation
+     * @see #post(Iterable, StreamObserver) for posing multiple messages at once
+     */
+    public final void post(T message, StreamObserver<IsSent> observer) {
+        checkNotNull(message);
+        checkNotNull(observer);
+        checkArgument(isNotDefault(message));
+
+        post(singleton(message), observer);
+    }
+
+    /**
+     * Posts the given messages to the bus.
+     *
+     * <p>Use the {@code Bus} class abstract methods to modify the behavior of posting.
+     *
+     * <p>The {@link StreamObserver} argument is the posting outcome observer;
+     * the {@link StreamObserver#onNext StreamObserver.onNext} will be called for each message
+     * passed to the bus.
+     *
+     * <p>A valid message will result in
```

Comment:
```
In case the message is accepted by the bus, {@linkplain IsSent IsSent} with the  {@link io.spine.base.Status.StatusCase#OK OK} status is passed to the observer.
```